oteapi/models/genericconfig.py

```diff
@@ -1,104 +1,125 @@
 """Generic data model for configuration attributes."""
-from typing import TYPE_CHECKING, Iterable, Mapping
+from typing import TYPE_CHECKING, Iterable, Mapping, MutableMapping
```

Comment:
Iterable, Mapping and MutableMapping are deprecated in typing and should now be imported from collections.abc